try-it.md

```diff
@@ -341,8 +281,8 @@ node-0   OK       provisioning          test1-md-0-m87bq       ipmi://192.168.11
 node-1   OK       provisioning          test1-controlplane-0   ipmi://192.168.111.1:6231   unknown            true
 ```
 
-You should be able to ssh into your host once provisioning is complete. See
-the libvirt DHCP leases to find the IP address for the host that was
+You should be able to ssh into your host once provisioning is completed. See
+the libvirt DHCP leases to find the IP address for the host that is now
```

Comment:
Not for v1a4 anymore, we do not use DHCP anymore to support properly the pivoting. The IP Addresses should be pretty much predictable (.100 for first master), or using the VIP (.249)

Reply:
Fixed